README.md

```diff
@@ -7,20 +7,14 @@
 This XBlock allows course authors to create and edit course content in Markdown
 and displays it as HTML.
 
-## Installation
-You may install the markdown-xblock using its setup.py, or if you prefer to use pip, run:
+## Installation with [Tutor](https://docs.tutor.edly.io)
+You may install the markdown-xblock to your Tutor environment by adding it to the `OPENEDX_EXTRA_PIP_REQUIREMENTS` list in `config.yml`:
 
-```shell
-pip install markdown-xblock==1.3.0
 ```
-
-If you prefer to install directly from Git, run:
-
-```shell
-pip install git+https://github.com/citynetwork/markdown-xblock.git@v1.3.0
+OPENEDX_EXTRA_PIP_REQUIREMENTS:
+- markdown-xblock==1.3.0
 ```
-
-You may specify the `-e` flag if you intend to develop on the repo.
+For additional information, please refer to the official [documentation](https://docs.tutor.edly.io/configuration.html#installing-extra-xblocks-and-requirements).
```

Comment:
I may be missing something, but to me it looks like the `installing-extra-xblocks-and-requirements` anchor does not exist on that page.

Reply:
You are right, I updated the link.